content/docs/command-reference/metrics/diff.md

```diff
@@ -58,6 +59,9 @@ lists all the current metrics without comparisons.
   useful when only one metrics file is in use or there is no intersection
   between the metric names.
 
+- `--precision <n>` - round metrics to n decimal digits. Rounds to 5 digits by
```

Comment:
💅 n -> `n`

Reply:
ideally, `Rounds to 5 digits default` should be very explicitly mentioned in the description

Reply:
I guess `n` -> `<n>` actually. At least that's how `git` does it.


> ideally, Rounds to 5 digits default should be very explicitly mentioned in the description

Could you reword, please? I didn't get your point.

Reply:
`I guess n -> <n> actually. ` - fine with either, just some highlighting is needed here it feels like

> Could you reword, please? I didn't get your point.

yeah, I think my point is if we always cut info at 5 decimal digits, it's better to mention in the description section above (not only here, in the `--precision` description).

Btw, do we apply `--precision` to all the columns? or only to the diff one? would be useful to be clear on this as well.

Reply:
@shcheklein Thanks! Adjusted.

Reply:
Good! thanks 🙏. I actually meant n in the `n decimal digits` part :)

Reply:
@shcheklein Sorry, I might be having a smooth-brain moment right now, but I don't get what you mean.

Reply:
:)

round metrics to n decimal digits

--->

round metrics to `n` decimal digits

Reply:
I've used `<n>`. But ok, using backticks now. Thanks!